packages/frontend/scss/message/_message.scss

```diff
@@ -385,36 +384,10 @@
   }
 }
 
-// Spec: container > 593px
-@media (min-width: 926px) {
```

Comment:
It seems we had 2 styles in media queries that were applied to all screens < 925px and also for screens > 926px
I'm not sure if that was for purpose.

@Simon-Laux seems it was introduced quite some time ago:
https://github.com/deltachat/deltachat-desktop/commit/5952a2058999f90303152718120d56c13316361f

Reply:
can't remember exactly. also it is older than the linked commit.